content/features/featuresDeepDive/lights/lights_introduction.md

```diff
@@ -61,6 +61,20 @@ that the full effect is achieved.
 const light = new BABYLON.HemisphericLight("HemiLight", new BABYLON.Vector3(0, 1, 0), scene);
 ```
 
+### The Rectangular Area Light
+
+A rectangular area light is defined by its position, width, and height. It emits light from the resulting surface towards the -Z direction. Even though the RectAreaLight class itself does not have a direction component, it can be attached to a transform node to be rotated and moved around.
+
+_A simple use of a rectangular area light_
+
+```javascript
+const light = new BABYLON.RectAreaLight("areaLight", new BABYLON.Vector3(0, 1, 0), 2, 2, scene);
+```
+
+One important thing to note is that, due to differences in implementation, StandardMaterial will reflect light based on its "roughness" value (instead of specular power, as with other lights). This difference in behavior is significant, and the scene should set the correct roughness value to achieve the desired look.

```

Comment:
I think this section could use a little more context. You mentioned the difference in the StandardMaterial roughness parameter, and mentioned that the scene should set this correctly. It seems like this could be expanded to describe if this process is completely automatic or if the user has any control over the roughness parameter. If the area light does not look right for specular reflections, is there anything to do to fix this value?

Reply:
Users will need to set roughness manually. However, we do have code in the exporters that converts specularPower to roughness and exporting StandardMaterial to GLTF. I have a PR in Babylon.js that tries to export that so users have access to it. When that PR gets merged I will update this documentation showing how to use that feature.